From 0ca334167ad710a1f9348355c59a1b601923c2f8 Mon Sep 17 00:00:00 2001
From: Zheng Junjie <zhengjunjie@iscas.ac.cn>
Date: Wed, 20 Nov 2024 16:12:34 +0800
Subject: [PATCH] set FETCHCONTENT_FULLY_DISCONNECTED=off

---
 debian/rules.em | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/rules.em b/debian/rules.em
index a856369..d04cf51 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -35,6 +35,7 @@ override_dh_auto_configure:
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DAMENT_PREFIX_PATH="@(InstallationPrefix)" \
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
+		-DFETCHCONTENT_FULLY_DISCONNECTED=off       \
 		$(BUILD_TESTING_ARG)
 
 override_dh_auto_build:
-- 
2.46.0

